src/repo_version.rs

```diff
@@ -206,6 +206,11 @@ impl worker::Runner<RepoVersionRequest> for Runner {
         if let Some(ref jira_session) = self.jira_session {
             if let Some(ref jira_config) = self.config.jira {
                 for config in &configs {
+                    // Don't run version scripts for jiras not mentioned
+                    if !jira::workflow::references_jira(&req.commits, &config.jira_project) {
```

Comment:
How would this manifest?

Reply:
It pulls up all the jira configs for that branch. which for like, main branch, would include all the jiras configured for that repo. as it is now, it wouldn't actually comment on all those repos, it would just run the version script unnecessarily.